src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Services/PermalinkServiceTests.cs

```diff
@@ -176,10 +181,20 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
                     subjectRepository,
                     tableBuilderService);
 
-                Assert.Equal(Guid.Parse(blobPath), result.Id);
+                Assert.Equal(expectedPermalinkId, result.Id);
                 Assert.InRange(DateTime.UtcNow.Subtract(result.Created).Milliseconds, 0, 1500);
                 Assert.Equal(PermalinkStatus.Current, result.Status);
             }
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
```

Comment:
I guess this means that the `data` user on Azure environments needs write access to this table.  Worth checking this out ahead of time - it'll also feed back into my db migrations on https://github.com/dfe-analytical-services/explore-education-statistics/pull/3530.

Reply:
Good point, since it's inserting into the `Permalinks` table I think we will need a minimum of 
`GRANT INSERT ON [dbo].[Permalinks] TO publicapp;`.

Prior to the read-replica work we've run these grants manually.

When this is merged in, depending on where we are with #3530 I will either create a faffy deploy ticket to run this grant or add it as a migration ticket.

I've left a comment on #3530 as I don't understand why this would be the `data` user (as things currently stand). The Data API app service uses the `content` db user `publicapp`.

Reply:
So maybe just get this PR in, and then I'll add this permission to the database migration for stage 1 replica.